library/FeeSheet.class.php

```diff
@@ -925,6 +925,23 @@ public function save(
             $main_supid = 0;
         }
 
+        function formatICD10Codes($inputArray): string
+        {
+            $result = [];
+
+            foreach ($inputArray as $item) {
+                // Check if the code_type is ICD10 and the code is set
+                if (isset($item['code_type'], $item['code']) && $item['code_type'] === 'ICD10') {
```

Comment:
hi @juggernautsei, I think this justifies the ICD10 entry in the billing table instead of the code being billed.

Reply:
I don't understand what you are saying. I can share with you that when the billing information is being saved from the LBF. It is showing justified when opening the fee sheet afterward.

Reply:
hi @juggernautsei, ran thru easy dev testing and it was justifying the ICD10 entry in the billing table instead of the code that needs to be justified

Reply:
If it is justified in the billing table, why it is not seen in the Fee Sheet when looking at it after saving an LBF?

Reply:
hi @juggernautsei, there is no supposition on my part. I view this PR as adding the feature to justify from an LBF. It looks like it needs more work to apply the created ICD10 string to the proper row/entry in the billing table.

Reply:
instead of this result:
![Screenshot from 2025-01-29 13-39-04](https://github.com/user-attachments/assets/994c5ed6-0567-4004-9d3f-c61c74adfff3)
how about this:
![Screenshot from 2025-01-29 13-39-43](https://github.com/user-attachments/assets/0edf6354-af67-4ade-9916-7e5c00766a03)

Reply:
@stephenwaite thanks for the review.